src/sections/collection/datasets-list/file-card/FileCard.module.scss

```diff
@@ -0,0 +1,74 @@
+@use 'sass:color';
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/typography.module";
+
+.container {
+  margin: 6px 0;
+  padding: 4px 10px;
+  border: 1px solid $dv-info-border-color;
```

Comment:
File cards in JSF version appear with another border color, this color is for datasets cards only.
Could be helpful to create the 3 color variables(collections, datasets and files) inside the design system  with names that relates to what they are specifically for.

Reply:
Sounds good, I added colors to the design system.